app/views/posts/show.html.erb

```diff
@@ -17,8 +17,13 @@
     <div class="post-paragraph">
         <p><%= @post.text %></p>
     </div>
+    <div class="delete-button-container">
+     <% if @post && can?(:destroy, @post) %>
+        <%= button_to "Delete Post", user_post_path(user_id: @user.id, id: @post.id), method: :delete, class: "delete-btn" %>
+      <% end %>
+    </div>
     <%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |f| %>
-      <span><%= f.submit "Like", class: "btn" %></span>
+      <span><%= f.submit "Like", class: "btn-delete" %></span>
     <% end %>
   </div>
 

```

Comment:
- [ ] Great work so far on adding the delete button for posts 👍 but it would be nice if you added a button for deleting the comment 🤔 because right now a user or an admin cannot delete the comments 😨 Kindly fix this issue to make your app work as expected 😊